src/components/generic/client.cpp

```diff
@@ -6,8 +6,8 @@
 #include "component/generic/v1/generic.grpc.pb.h"
 
 ProtoType GenericClient::do_command(std::unordered_map<std::string, ProtoType*> command) {
-    viam::component::generic::v1::DoCommandRequest req;
```

Comment:
Note to reviewers: this change (and analogous changes) are unrelated to the core PR, but were necessary to match updates to our protos.